callstack/react-native-builder-bob
# fix: correct the paths for codegen generated header files on views

### Summary

Closes #669

We were trying to import the codegen-generated view scaffold code for iOS from the `react` target. And since we now generate the codegen scaffold code for the user, it's placed under the same pod as the user's code lives. This means we can now get rid of the import path and just import using the header name.

> Note: This will cause problems for users who don't ship their codegen-compiled code. Since this is not a path we want to support, there is no compatibility logic. If these users wish to go without shipping their codegen-compiled code, then they need to add those import paths again. The import paths are described in the [Official React Native documentation](https://reactnative.dev/docs/fabric-native-components-introduction?platforms=ios#2-building-your-native-code)

### Test plan

1. Generate a new view library that supports the new architecture
2. Install the pods and trigger a build on iOS
3. Make sure the example app bulids and runs fine.
